package.json

```diff
@@ -80,12 +82,10 @@
       }
     },
     "nonRenderableFiles": [
```

Comment:
because of previous discussion, this should be removed entirely (from readme to if it is described there) because at the end it doesn't matter if we have here those files or not, it has no impact on generation

Reply:
It has no impact on generation, but if I removed `styles.css` then I won't have a styles for html. In React case each file if hasn't `.js` extension is skipped, it means that it doesn't performed -> and then copied, so it must be inside `nonRenderableFiles` array. In Nunjucks it works, because Nunjucks can render all extensions -> in Nunjucks `css` is rendered and then copied to output.

Reply:
oh, wait, so basically, every non-js file has to be put there? not cool 🤔

Reply:
> oh, wait, so basically, every non-js file has to be put there? not cool 🤔

Any files which needs to be copied as is yes 😄

Reply:
it is a lot of configuration and is error-prone. Just look at [this](https://github.com/asyncapi/nodejs-template/tree/master/template) example and see how much files I will have to maintain in the configuration :( 

imho we need to improve this, don't you think (not for this PR of course)

Reply:
We have an issue to simplify Generator https://github.com/asyncapi/generator/issues/464 and we should take care about this case in development. At the moment Generator works in this way as I described and it wasn't in scope of `React engine` task, I mean changing way of copied files in React has impact on code and for that we didn't that.

Reply:
ok, so just add there a comment that when working on refactoring we need to improve this and that is it, or create a separate issue if you want

Reply:
Done https://github.com/asyncapi/generator/issues/464#issuecomment-747552843

Reply:
I created separate issue for that, because it has a higher priority than refactoring, it is bug.

Reply:
nah, not a bug, in my opinion, it works as designed, just needs improvement. It is nothing comparing to the great job you did with react engine guys!